src/dcTMD/dcTMD.py

```diff
@@ -50,22 +92,22 @@ class WorkEstimator(TransformerMixin, BaseEstimator):
         bootstrapping errors are calculated.
     s_W_mean_ :
         Bootstrapping error of the mean work. Calculated via
-        estimate_free_energy_errors().
+        `estimate_free_energy_errors()`.
```

Comment:
I guess I would use here (and the lines below) the mkdocs syntax [dcTMD.method][] to get automated links inside the docs.

Reply:
Thanks, fixed it now!